.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: csharp
+
+os: linux
+dist: trusty
+group: edge
+sudo: required
+mono: latest
+dotnet: 2.0.0
+
+deploy:
+  provider: script
+  script:
+  - nuget pack Syncromatics.Engineering.DX.Common.nuspec -Version $TRAVIS_TAG
```

Comment:
Will using nuget pack with a nuspec require you to define the dependencies in the nuget file as well as the csproj?

Reply:
This nuspec is purely for the template packaging so it can be used with `dotnet new`. It is not part of any template documents that should end up in any other repo. As such, this nuspec does not have any dependencies on anything else.

Reply:
Oh I see. It's also a pretty good example for packing pretty much any other of our open source projects (with dotnet pack instead of nuget), but if it's not intended for that then it doesn't matter.